app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_secure_password
+  validates_presence_of :name, :password
+  validates_uniqueness_of :name
```

Comment:
Should we have an email just in case someone has a the same name? and validate that uniqueness instead of name.